src/openzaak/components/zaken/tests/test_zaken.py

```diff
@@ -574,3 +577,131 @@ def test_zaak_archiefactiedatum_afleidingswijze_ingangsdatum_besluit(self):
         self.assertEqual(
             zaak.archiefactiedatum, date(2030, 5, 3)  # 2020-05-03 + 10 years
         )
+
+
+@tag("external-urls")
+class ZaakCreateExternalURLsTests(JWTAuthMixin, APITestCase):
```

Comment:
can you add a test with an external zaaktype that is not published (`concept=True`)?

Reply:
testcase is added